custom/icds/tasks.py

```diff
@@ -155,27 +156,31 @@ def delete_old_images():
         start = datetime.utcnow()
         max_age = start - timedelta(days=90)
```

Comment:
Where did the 90 day threshold come from?

Reply:
It's been part of the system design drive the beginning. It's in all the system sizing docs.